app/src/main/java/org/qosp/notes/data/repo/NotebookRepository.kt

```diff
@@ -16,7 +18,7 @@ class NotebookRepository(
 
     suspend fun delete(vararg notebooks: Notebook) {
         val affectedNotes = notebooks
-            .map { noteRepository.getByNotebook(it.id).first() }
+            .map { noteDao.getByNotebook(it.id, defaultOf()).first() }
```

Comment:
The notes should be updated as well - the `notebookId` should be set to null in the database